hoomd/variant.py

```diff
@@ -73,6 +73,14 @@ def __init__(self, value):
         Variant.__init__(self)
         _hoomd.VariantConstant.__init__(self, value)
 
+    def __eq__(self, other):
+        """Return whether two variants are equivalent."""
```

Comment:
These `__eq__` methods are highly redundant. I wonder if you can do something less redundant, by defining the relevant attributes in each subclass and only implementing this `__eq__` once in the parent class?